packages/frontend/editor-ui/src/components/RunData.vue

```diff
@@ -810,7 +814,100 @@ function getNodeHints(): NodeHint[] {
 
 	return [];
 }
+function getConsolidatedHint(): { message: string } | null {
```

Comment:
This could be a computed that always returns `string[]` (array of messages)
and then you could have another computed that converts the `string[]` to `string` for a consolidated message (using Intl.ListFormat)